ARCHES

```diff
@@ -1,3 +1,6 @@
+i386-apple-darwin
+x86_64-apple-darwin
+aarch64-apple-darwin
```

Comment:
Sorry for the confusion, `darwin` tuples are not currently built using the `ARCHES` pattern.  Currently the `ARCHES` list builds only artifacts that can be cross compiled using a linux build system.

The gem build process currently calls the following in a macOS VM with Xcode available.
```
make TARGET=x86_64-apple-darwin
make TARGET=arm-iphone-darwin
make TARGET=aarch64-iphone-darwin
```

I do like the idea of having all supported tuples in the `ARCHES` file however the `Makefile` would need to be updated to either detect the running OS and skip `darwin` when run on a linux platform or `darwin` tuples need to be a separate file.

Reply:
Note if removing is the path the other `darwin` added targets for `iphone` at the end of the file are also not compilable on `Linux`.